src/schema/feeds.ts

```diff
@@ -1,4 +1,14 @@
-import { FeedAdvancedSettings, AdvancedSettings } from '../entity';
+import {
```

Comment:
@rebelchris let me know what you think of the changes here. Or do you want to have a subscription on the alerts myFeed property changes?

Reply:
This seems like it could trigger many, many times?
Not sure on the implication maybe the less friction for now would be listening on alerts.

Not particular against this solution, but maybe let's ask @idoshamun if he sees any issue with this?

Reply:
Good point on the frequency which would make it somehow ideal to be listening on alerts.

Reply:
You don't want to add anything unnecessary for the feed call. It may have a tremendous impact.
We already have a mechanism to identify when the user enable my feed, let's use it.

Reply:
Yep. I just realized we used the `alerts.filter` for this exact case, when the user updated their filters. We can probably do the inline direct from the updateAlerts mutation.